src/pocketmine/item/Item.php

```diff
@@ -524,6 +542,27 @@ public function getEnchantment(int $id){
 
 		return null;
 	}
+	
+	/**
+	* @param int $id
+	* @param int $level
+	*/
+	public function removeEnchantment(int $id, int $level = null){
```

Comment:
Not gonna change. It's possible to multi enchant an item. If you know how nbt tags work, you can easily add enchants of the same ID, but different levels.